src/main/java/uk/ac/ebi/eva/pipeline/io/mappers/exceptions/VCFRecordWithNonVariantSites.java

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright 2015-2017 EMBL - European Bioinformatics Institute
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package uk.ac.ebi.eva.pipeline.io.mappers.exceptions;
+
+/**
+ * This exception is thrown when the data from a record on a VCF file contains non-variant sites
+ */
+public class VCFRecordWithNonVariantSites extends Exception {
```

Comment:
I would change this to `NonVariantException`, as "sites" is misleading. a VCF record doesn't have sites. it has genotypes. And change the message to "The VCF record doesn't report any variant".